src/app/shared/text-with-links/text-with-links.component.ts

```diff
@@ -30,4 +30,8 @@ export class TextWithLinksComponent {
     }
     this.parts = parts;
   }
+
+  public normaliseHref(href: string): string {
+    return href.replace(/[\.]$/, '');
```

Comment:
I think we should also drop comma's. So `/[\.,]$/`

You can also add a unit test for that.